flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java

```diff
@@ -700,6 +700,15 @@ private static void setupAlarm(
       AlarmManager alarmManager,
       long epochMilli,
       PendingIntent pendingIntent) {
+
+    if (notificationDetails.scheduleMode == null) {
+      // This is to account for notifications created in older versions prior to allowWhileIdle
+      // being added so the deserialiser.
+      // Reference to old behaviour:
+      // https://github.com/MaikuB/flutter_local_notifications/blob/master/flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java#L735
+      notificationDetails.scheduleMode = ScheduleMode.exact
```

Comment:
Code's not compiling as you're missing a semicolon here

Reply:
I am so sorry. No idea how I missed a semicolon. I have pushed the changes.

Reply:
Thanks. I've used your PR to fix existing typos I had in the comments too